js/runner_test.go

```diff
@@ -765,7 +765,7 @@ func TestVURunInterruptDoesntPanic(t *testing.T) {
 					assert.Contains(t, vuErr.Error(), "context canceled")
 				}()
 				<-ch
-				time.Sleep(time.Millisecond * 1) // NOTE: increase this in case of problems ;)
+				time.Sleep(time.Microsecond * 1) // NOTE: increase this in case of problems ;)
```

Comment:
Why not reducing it if it works correctly? In fact, I've seen the flakiness induced by this test is lower with a lower value 🤷🏻